srcpkgs/python3-pendulum/template

```diff
@@ -14,6 +15,12 @@ changelog="https://github.com/sdispater/pendulum/blob/master/CHANGELOG.md"
 distfiles="https://github.com/sdispater/pendulum/archive/refs/tags/${version}.tar.gz"
 checksum=4185efa48358cb9dca57a6afbc471d4b260e70f56609d5030197373564ac4a07
 
+do_build() {
+	mkdir -p build
+	TMPDIR="${PWD}/build" python3 -m pip wheel --no-deps --no-clean \
+		--use-pep517 --no-build-isolation --wheel-dir dist .
+}
```

Comment:
This one is great. `python -m build` attempts to run poetry's undocumented `build.py` plugin.

I've always been pretty meh about the use of `python -m` anything unless the goal is very specifically to inject $PWD into the python module search path, which granted you may sometimes want to do (pytest, specifically).

`/usr/bin/pyproject-build` arguably works just as well.

Reply:
Thanks for catching this; it completely escaped me!